tests/unit/backend/layers/common/base_api_test.py

```diff
@@ -24,6 +24,10 @@ def setUp(self):
         )
         self.mock_config.start()
 
+        # TODO: this can be improved, but the current authorization method requires it
+        self.mock = patch("backend.common.corpora_config.CorporaAuthConfig.__getattr__", return_value="mock_audience")
+        self.mock.start()
```

Comment:
Interesting. This wasn't there before; I just moved it to a different class. But I'll add, sure.

Reply:
I'm actually just not going to touch this since I won't be messing with mock stuff at all with the new approach in the new PR (coming imminently). Let me know if you think this is an actual problem for the existing tests, as it was.